src/Chapter3.hs

```diff
@@ -480,6 +533,63 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city totally.
 -}
 
+data Castle   = Castle  String                          deriving (Show)
+data Wall     = Wall                                    deriving (Show)
+data ChOrLib  = Church  | Library                       deriving (Show)
+data House    = HouseP1 | HouseP2 | HouseP3 | HouseP4   deriving (Show)
```

Comment:
I understand that the number of people in a house is limited t o1, 2, 3, and 4. I revised the code. HousePn means the house with n people.